tests/integ/modin/series/test_dt_accessor.py

```diff
@@ -433,6 +433,71 @@ def test_days_in_month(property):
     )
 
 
+@sql_count_checker(query_count=1)
+@pytest.mark.parametrize(
+    "date_format",
+    [
+        "a%d-%m-%Y-%H-%M-%S-%f-%j-%X-%%b",
+        "%d-%m-%Y-%H-%M-%S-%f-%j-%X-%%b",
+        "a%d-%m-%Y-%H-%M-%S-%f-%j-%X-%%",
+        "%d-%m-%Y-%H-%M-%S-%f-%j-%X-%%",
```

Comment:
can you add a couple cases for more pathological-looking strings, like `"%%%M"` or `"%%M"`? Also, what happens if we have an unfinished expression like `"abc%"`?

Reply:
Added.

Reply:
For the `"abc%"` case, it will be kept as is. So it's treated the same way as `"abc%%"`. This is how pandas behaves and we now have a test for it too.